docs/en/Overview.md

```diff
@@ -44,27 +41,15 @@ from over 200 institutions including
 [Tencent](https://www.tencent.com),
 [UC Berkeley](https://amplab.cs.berkeley.edu/),
 and [Yahoo](https://www.yahoo.com/).
-The project is the data layer of the Berkeley Data Analytics Stack ([BDAS](https://amplab.cs.berkeley.edu/bdas/))
-and is also part of the [Fedora distribution](https://fedoraproject.org/wiki/SIGs/bigdata/packaging).
+
 Today, Alluxio is deployed in production by
 [hundreds of organizations](https://www.alluxio.org/community/powered-by-alluxio)
-and runs on clusters exceeding 1,000 nodes.
+with largest deployment exceeding 1,000 nodes.
```

Comment:
I would probably say "with the largest deployments" or "with the largest deployment"